notebooks/official/model_monitoring/get_started_with_model_monitoring_xgboost.ipynb

```diff
@@ -1217,62 +1216,1261 @@
+        "      Open in Vertex AI Workbench\n",
+        "    </a>\n",
+        "  </td>\n",
+        "</table>\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "lA32H1oKGgpf"
+      },
+      "source": [
+        "## Overview\n",
         "\n",
-        "Otherwise, you can delete the individual resources you created in this tutorial."
+        "This tutorial demonstrates how to use `Vertex AI Model Monitoring` for XGBoost models.\n",
+        "\n",
+        "Learn more about [Vertex AI Model Monitoring](https://cloud.google.com/vertex-ai/docs/model-monitoring)."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "de998a3953c9"
+      },
+      "source": [
+        "### Objective\n",
+        "\n",
+        "In this notebook, you learn to use the `Vertex AI Model Monitoring` service to detect feature skew and drift in the input predict requests for XGBoost models.\n",
+        "\n",
+        "This tutorial uses the following Google Cloud ML services:\n",
+        "\n",
+        "- `Vertex AI Model Monitoring`\n",
+        "- `Vertex AI Prediction`\n",
+        "- `Vertex AI Model` resource\n",
+        "- `Vertex AI Endpoint` resource\n",
+        "\n",
+        "The steps performed include:\n",
+        "\n",
+        "- Download a pre-trained XGBoost model.\n",
+        "- Upload the pre-trained model as a `Model` resource.\n",
+        "- Deploy the `Model` resource to the `Endpoint` resource.\n",
+        "- Configure the `Endpoint` resource for model monitoring:\n",
+        "  - drift detection only -- no access to training data.\n",
+        "  - predefine the input schema to map feature alias names to the unnamed array input to the model.\n",
+        "- Generate synthetic prediction requests for drift.\n",
+        "\n",
+        "\n",
+        "Learn more about [Introduction to Vertex AI Model Monitoring](https://cloud.google.com/vertex-ai/docs/model-monitoring/overview)."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "edba71dc9840"
+      },
+      "source": [
+        "### Model\n",
+        "\n",
+        "The model used for this tutorial is a pretrain XGBoost model that was trained on the [Iris dataset](https://www.tensorflow.org/datasets/catalog/iris) from [TensorFlow Datasets](https://www.tensorflow.org/datasets/catalog/overview). The trained model predicts the type of Iris flower species from a class of three species: setosa, virginica, or versicolor."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "t6Cd51FkG09E"
+      },
+      "source": [
+        "### Costs \n",
+        "\n",
+        "This tutorial uses billable components of Google Cloud:\n",
+        "\n",
+        "* Vertex AI\n",
+        "* BigQuery\n",
+        "* Cloud Storage\n",
+        "\n",
+        "Learn about [Vertext AI\n",
```

Comment:
put on one line:

"Learn about [Vertext AI\pricing](https://cloud.google.com/vertex-ai/pricing) and [Cloud Storage\n", 

Want to make sure the link works. Do the same for [Cloud Storage... AND [Pricing... (now in line 1373). Don't want to break up the link text and the corresponding URL.